### PR TITLE
I finished pnorm, qnorm function, please review.

### PR DESCRIPTION
**Pull request by czwcandy**

### CODE REVIEW

File: src/MathDistOps/pnorm.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    0.8413447460685429
+
+    Graph:
+    --------
+    - The blue curve represents the normal distribution CDF for the given parameters.
+    - The vertical red line corresponds to the specified value `x`.
+
+    Example Usage:
+    ```python
+    import matplotlib.pyplot as plt
```
Comment (sandygross) [inline src/MathDistOps/pnorm.py]: Altair or matplotlib??

**Review by sandygross — DISMISSED** [review]

**Comment by sivakornchong** [comment]
Can I also request for the docstring to follow the numpydoc style format? It is something that we are more familiar with in MDS? Something like this.. 

Parameters
----------
word_counts : collections.Counter
Counter object of word counts.
n : int, optional
Plot the top n words. By default, 10.

File: src/MathDistOps/pnorm.py (hunk 1 above)
Comment (czwcandy) [reply src/MathDistOps/pnorm.py]: matplotlib though :)

**Review by sivakornchong — APPROVED** [review]
Looks good. Comments have been incorperated.

**Pull request merged**